### PR TITLE
Trade Post Remake troi

### PR DESCRIPTION
**Pull request by Erikafox**
## About The Pull Request
Remasters the currently removed and unpersoned wasteplanet tradepost map. You have never seen this pr before.
dreamseeker_aqYq8vThLj
StrongDMM-2024-10-17 17 39 28
StrongDMM-2024-10-17 17 40 03

## Why It's Good For The Game
A 'new' ruin of decent quality and some more mapgen options should enrich the experience.

## Changelog

:cl: Erika Fox
add: An abandoned tradepost has been located in the system. Investigate nearby waste worlds for more information.
add: preburnt flares
/:cl:

### CODE REVIEW

File: code/datums/mapgen/planetary/WasteGenerator.dm [hunk 1]
```diff
@@ -66,9 +66,9 @@
 		BIOME_COLD_CAVE = list(
 			BIOME_LOWEST_HUMIDITY = /datum/biome/cave/waste,
 			BIOME_LOW_HUMIDITY = /datum/biome/cave/waste/rad,
-			BIOME_MEDIUM_HUMIDITY = /datum/biome/cave/waste,
-			BIOME_HIGH_HUMIDITY = /datum/biome/cave/waste/rad,
```
Comment (thgvr) [inline code/datums/mapgen/planetary/WasteGenerator.dm]: error?

File: _maps/shuttles/syndicate/syndicate_cybersun_kansatsu.dmm [hunk 2]
```diff
@@ -1,20 +1,4 @@
 //MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
```
Comment (thgvr) [inline _maps/shuttles/syndicate/syndicate_cybersun_kansatsu.dmm]: Not sure if this is intended, but it needs to not be in this PR.

File: code/modules/clothing/outfits/factions/solgov.dm [hunk 3]
```diff
@@ -161,7 +161,6 @@
 		/obj/item/flashlight/seclite=1,\
 		/obj/item/kitchen/knife/combat/survival=1,\
 		/obj/item/stack/marker_beacon/ten=1)
-
```
Comment (thgvr) [inline code/modules/clothing/outfits/factions/solgov.dm]: ?

File: code/modules/mining/equipment/marker_beacons.dm [hunk 4]
```diff
@@ -152,3 +158,25 @@ GLOBAL_LIST_INIT(marker_beacon_colors, sortList(list(
 	if(input_color)
 		picked_color = input_color
 		update_appearance()
+
+/obj/item/stack/marker_beacon/acidproof
```
Comment (thgvr) [inline code/modules/mining/equipment/marker_beacons.dm]: Take this to another PR unless it's completely necessary

File: code/modules/mob/living/simple_animal/corpse.dm [hunk 5]
```diff
@@ -251,3 +251,33 @@
 	id_job = "SolGov Sonnensoldner"
 	outfit = /datum/outfit/job/solgov/sonnensoldner
 	id_access_list = list(ACCESS_SOLGOV)
+
+//inteq
+
+/obj/effect/mob_spawn/human/corpse/inteq
```
Comment (thgvr) [inline code/modules/mob/living/simple_animal/corpse.dm:275]: I don't really want stocked corpses in a ruin like this.

**Review by thgvr — DISMISSED** [review]
This PR has far too many unrelated changes

File: _maps/shuttles/syndicate/syndicate_cybersun_kansatsu.dmm (hunk 2 above)
Comment (Erikafox) [reply _maps/shuttles/syndicate/syndicate_cybersun_kansatsu.dmm]: The branch had some fucky bullshit I tried to do with beacons in it's first iteration and that was a consequence of trying to untangle it.

File: code/modules/mob/living/simple_animal/corpse.dm (hunk 5 above)
Comment (Erikafox) [reply code/modules/mob/living/simple_animal/corpse.dm:275]: what would you suggest trimming it down to?

File: code/datums/mapgen/planetary/WasteGenerator.dm (hunk 1 above)
Comment (thgvr) [reply code/datums/mapgen/planetary/WasteGenerator.dm]: I misread it, regardless the biome changes should be in a different PR.

File: code/game/turfs/open/floor/plating/wasteplanet.dm [hunk 6]
```diff
@@ -32,31 +32,56 @@
...
+	icon_state = pick(list(
+		"conc_smooth",
+		"conc_slab_1",
+		"conc_slab_2",
+		"conc_slab_3",
+		"conc_slab_4",
+		"conc_tiles"
+	))
+
+/turf/open/floor/concrete/wasteplanet/MakeSlippery(wet_setting, min_wet_time, wet_time_to_add, max_wet_time, permanent)
```
Comment (FalloutFalcon) [inline code/game/turfs/open/floor/plating/wasteplanet.dm:67]: why is specifically wasteplanet concreate not slippery

Comment (Erikafox) [reply code/game/turfs/open/floor/plating/wasteplanet.dm:67]: wasteplanets can spawn with water vapor in the atmosphere (unless this has been changed. I did not check), and having a biome turn into a giant slip and slide doesn't sound like an engaging gameplay experience.

**Review by meemofcourse — DISMISSED** [review]
imagen
no

imagen
tone down the engi gear ok thanks

i don't know how i feel about corpses with inteq gear but that's not a concern for this pr

**Comment by meemofcourse** [comment]
hey can you fix your checks

**Review by Gristlebee — COMMENTED** [review]
Feedback after testing on live:
---
The main office building is exposed to the waste planet atmos. Either give it a proper door or replace the tiles inside with tile subtypes with planet atmos.
image
There's not really much in these rooms to justify them being bolted, and consequently being worth the effort to break inside. I feel it would good to add some kind of minor loot or money in the office.
image
The paths are fairly plain looking. I think they would benefit from some decals or siding.
image
There's a normal NT Commander here instead of an Inteq Commissioner here. You can cut this one since there's already two other commissioners to loot.
image
.

I'd suggest swapping the position of the mongrel with one of the commissioners. From a flavor standpoint it makes more sense for it be near it's mags, and players have to go through a bit more effort to get the stronger gun.

Players had some confusion on whether the Tradepost was fully Inteq owned. If they were contracted, I think it would be good to add some civilian corpses around the base.

The outer area of the ruin is very dark for what was supposed to be an inhabited area. There's only a single broken floodlight for what would have been a cargo port. I'd suggest scattering around some more broken or maybe even a few working lights on the exterior of the buildings.

As for the Inteq corpses, you could make a subtype of the outfits for use in the ruin with some of their gear cut. The main concerns to me are the corpsman and the vanguard. The corpsman comes with a loaded webbing, so you can either cut it or give them an empty one. Cut the gear from the Vanguard corpse's pockets, and the telebaton in their backpack.

**Comment by Erikafox** [comment]
addressed review (aside from the roads. idk how to get a nice decal pattern on those and think that the plain is somewhat alright)

**Pull request merged**